recipes/mdnsd/mdns_544.oe

```diff
@@ -76,16 +82,9 @@ pkg_prerm_${PN} () {
         -i $D/etc/nsswitch.conf
 }
 
-FILES_${PN} += "${libdir}/libdns_sd.so.1 \
-                ${bindir}/dns-sd \
-                ${libdir}/libnss_mdns-0.2.so \
-                ${sysconfdir}/nss_mdns.conf"
-
-FILES_${PN}-dev += "${libdir}/libdns_sd.so \
-                    ${includedir}/dns_sd.h "
-
-FILES_${PN}-man += "${mandir}/man8/mdnsd.8 \
-                    ${mandir}/man5/nss_mdns.conf.5 \
-                    ${mandir}/man8/libnss_mdns.8"
-
-PACKAGES = "${PN} ${PN}-dev ${PN}-man ${PN}-dbg"
+FILES_${PN}-libnss-mdns-dev = ""
+FILES_${PN}-dev = ""
+FILES_${PN}-libnss-mdns += "${libdir}/libnss_mdns-0.2.so ${sysconfdir}/nss_mdns.conf"
+FILES_${PN}-libdns-sd-dev = "${includedir}/dns_sd.h"
+FILES_${PN}-libdns-sd += "${libdir}/libdns_sd.so"
```

Comment:
we usually put the symlink in the -dev package, and the versionen libs in the lib package. any specific reason for not doing the same here?